modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -81,74 +91,76 @@
 			<#list entries as entry>
 				<#if entry?has_content>
 					<#assign
-						portalURL=portalUtil.getLayoutURL(themeDisplay)
-						productId=entry.getClassPK() + 1
-						product=restClient.get("/headless-commerce-admin-catalog/v1.0/products/" + productId + "?nestedFields=productSpecifications,attachments" )
-						productAttachments=product.attachments![]
+						portalURL = portalUtil.getLayoutURL(themeDisplay)
+						productId = entry.getClassPK() + 1
+						product = restClient.get("/headless-commerce-admin-catalog/v1.0/products/" + productId + "?nestedFields=productSpecifications,attachments")
+						productAttachments = product.attachments![]
 						productCategories=product.categories![]
-						productDescription=stringUtil.shorten(htmlUtil.stripHtml(product.description.en_US!""), 150, "..." )
-						productSpecifications=product.productSpecifications![]
-						productURL=portalURL?replace("solutions-marketplace", "p" ) + "/" + product.urls.en_US />
+						productDescription = stringUtil.shorten(htmlUtil.stripHtml(product.description.en_US!""), 150, "...")
+						productSpecifications = product.productSpecifications![]
+						productURL = portalURL?replace("solutions-marketplace", "p") + "/" + product.urls.en_US
+					/>
 
 					<a class="solution-search-results-card bg-white border-radius-medium d-flex flex-column mb-0 rounded text-dark text-decoration-none" href=${productURL}>
-						<div class="align-items-center d-flex image-container justify-content-center">
+						<div class="align-items-center d-flex image-container justify-content-center mb-3">
 							<img
 								alt=${product.name.en_US}
-								class="h-100 mw-100 rounded"
-								src="${product.thumbnail}" />
+								class="solution-search-image rounded"
+								src="${product.thumbnail}"
+							/>
 						</div>
 
-						<div class="align-items-center card-image-title-container d-flex pb-3">
+						<div class="align-items-center card-image-title-container d-flex">
 							<div class="pl-2">
-								<div class="font-weight-semi-bold h2 mt-1">
-									${product.name.en_US}
-								</div>
-
 								<#if productSpecifications?has_content>
 									<#assign productPriceModels = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
 
 									<#list productPriceModels as productPriceModel>
-										<div class="color-neutral-3 font-size-paragraph-small mt-1">
+										<div class="developer-name font-size-paragraph-small">
 											${productPriceModel.value.en_US}
 										</div>
 									</#list>
 								</#if>
+
+								<div class="font-weight-semi-bold h2 mt-1">
+									${product.name.en_US}
```

Comment:
You need to validate each variable used on this page. What happens if a variable returns a null value? Run the script migration to make sure what you've developed works, or contact me to run some tests in the LXC environment.